virt/virt-2-6-release-notes.adoc

```diff
@@ -147,11 +138,13 @@ data:
 ----
 <1> Replace `<cpu-model>` with the actual CPU model value. You can determine this value by running `oc describe node <node>` for all nodes and looking at the `cpu-model-<name>` labels. Select the CPU model that is present on all of your nodes.
 
+// No bug attached to this -- is it still relevant?
+
 * {VirtProductName} cannot reliably identify node drains that are triggered by running either `oc adm drain` or `kubectl drain`. Do not run these commands on the nodes of any clusters where {VirtProductName} is deployed. The nodes might not drain if there are virtual machines running on top of them.
 
 ** The current solution is to xref:../virt/node_maintenance/virt-setting-node-maintenance.adoc#virt-setting-node-maintenance[put nodes into maintenance].
```

Comment:
@stu-gott Do you know if this should be retained for 2.6?

Reply:
This release note can be dropped for 2.6. We now honor node drains.